app/views/handlers/__init__.py

```diff
@@ -38,3 +39,8 @@ def individual_response_url(
                 journey=journey,
             )
     return None
+
+
+def show_individual_response(location, schema):
+    section = schema.json.get("individual_response", {}).get("individual_section_id")
```

Comment:
This method would be better placed in questionnaire_schema class.

Reply:
This call to individual_section_id is also in individual_response and should be refactored to use the common method mentioned above.

Reply:
As Ian suggest, fetching the section id can be moved to questionnaire schema. And then we should update individual_response handler to use that new method. currently it is:
```py
@cached_property
    def individual_section_id(self):
        return self._schema.json.get("individual_response", {}).get(
            "individual_section_id"
        )
```

Reply:
This is now using that new method, thanks!